docs/make.jl

```diff
@@ -2,76 +2,76 @@ using Documenter
+
 
-open(joinpath(@__DIR__, "src", "contributing.md"), "w") do io
-    # Point to source license file
-    println(io,
-            """
-            ```@meta
-            EditURL = "https://github.com/trixi-framework/TrixiBottomTopography.jl/blob/main/CONTRIBUTING.md"
-            ```
-            """)
-    # Write the modified contents
-    for line in eachline(joinpath(dirname(@__DIR__), "CONTRIBUTING.md"))
-        line = replace(line, "[LICENSE.md](LICENSE.md)" => "[License](@ref)")
-        line = replace(line, "[AUTHORS.md](AUTHORS.md)" => "[Authors](@ref)")
-        println(io, line)
-    end
-end
+# Copy contributing information to not need to synchronize it manually
+contributing_text = read(joinpath(dirname(@__DIR__), "CONTRIBUTING.md"), String)
+contributing_text = replace(contributing_text,
+                            "[LICENSE.md](LICENSE.md)" => "[License](@ref)")
+contributing_text = replace(contributing_text,
+                            "[AUTHORS.md](AUTHORS.md)" => "[Authors](@ref trixi_bt_authors)")
+write(joinpath(@__DIR__, "src", "contributing.md"), contributing_text)
+
+# Copy contents form README to the starting page to not need to synchronize it manually
+readme_text = read(joinpath(dirname(@__DIR__), "README.md"), String)
+readme_text = replace(readme_text,
+                      "[LICENSE.md](LICENSE.md)" => "[License](@ref)")
+readme_text = replace(readme_text,
+                      "[AUTHORS.md](AUTHORS.md)" => "[Authors](@ref trixi_bt_authors)")
+readme_text = replace(readme_text,
+                      "<p" => "```@raw html\n<p")
+readme_text = replace(readme_text,
+                      "p>" => "p>\n```")
+readme_text = replace(readme_text,
+                      r"\[comment\].*\n" => "")    # remove comments
+write(joinpath(@__DIR__, "src", "home.md"), readme_text)
 
 # Make documentation
-makedocs(
-    sitename = "TrixiBottomTopography.jl",
-    format = Documenter.HTML(
-        # Set canonical URL to GitHub pages URL
-        canonical = "https://trixi-framework.github.io/TrixiBottomTopography.jl/dev"
-    ),
-    modules = [TrixiBottomTopography],
-    pages = [
-        "Home" => "index.md",
-        "Overview" => [
-            "Data conversion" => "conversion.md",
-            "B-spline structure" => "structure.md",
-            "B-spline function" => "function.md",
-        ],
-        "Trixi.jl examples" => "Trixi.md",
-        "Reference" => "reference.md",
-        "Authors" => "authors.md",
-        "Contributing" => "contributing.md",
-        "Code of Conduct" => "code_of_conduct.md",
-        "Licence" => "licence.md"
-    ],
-    strict = true # to make the GitHub action fail when doctests fail
-)
+makedocs(;
+         modules = [TrixiBottomTopography],
+         authors = "Andrew R. Winters <andrew.ross.winters@liu.se>, Michael Schlottke-Lakemper <michael@sloede.com>",
+         repo = "https://github.com/trixi-framework/TrixiBottomTopography.jl/blob/{commit}{path}#{line}",
```

Comment:
According to the [Documenter.jl docs](https://documenter.juliadocs.org/stable/lib/remote-links/#Documenter.Remotes.GitHub), this should be written like so, I believe:
```suggestion
         repo = GitHub("trixi-framework", "TrixiBottomTopography.jl"),
```

Reply:
Trixi.jl and others seem to not have this line, so I think I will just remove it.